faunadb/client_test.go

```diff
@@ -2382,6 +2382,112 @@ func (s *ClientTestSuite) TestEchoQuery() {
 	s.Require().True(firstSeen > 0 && s.client.GetLastTxnTime() >= firstSeen)
 }
 
+func (s *ClientTestSuite) TestPaginateAccessProviders() {
+	var obj f.ObjectV
+	var val f.Value
+	var arr f.Arr
+	var b bool
+
+	name := f.RandomStartingWith("name_")
+	issuer := f.RandomStartingWith("https://no.issuer.fauna.com/")
+	jwksUri := f.RandomStartingWith("https://validuri.com/")
```

Comment:
`https://validuri.com/` is actually not a valid url, you should use the pattern `https://xxxx.auth0.com`, because Auth0 is the only provider we support at the moment.